cli/get.go

```diff
@@ -107,9 +107,6 @@ func mainGet(ctx *cli.Context) error {
 		ListFlat:      ctx.Bool("list-flat"),
 		ListPrefix:    ctx.String("prefix"),
 	}
-	if b.ListExisting && !ctx.IsSet("objects") {
-		b.CreateObjects = 0
-	}
```

Comment:
this is not needed since `--objects` if specified does the actual limit, we do not have to zero out CreateObjects() when `--objects` is not set it is zero anyway.

Reply:
ok